pkg/controller/eventtype/reconcile_test.go

```diff
@@ -43,6 +46,10 @@ const (
 
 	fn1 = "fake-feed-name-1"
 	fn2 = "fake-feed-name-2"
+
+	listError   = "unexplained test error listing"
```

Comment:
bit of a nit, but since these are test induced, consider making it obvious that these are created by the test. Something like:
"test induced error listing"

or something like that.
Just thinking that if a test fails and you look at the logs, might not be obvious that this is actually something that should happen.

Reply:
Done.